apps/journeys-admin/src/components/JourneyVisitorsList/FilterDrawer/FilterDrawer.tsx

```diff
@@ -0,0 +1,121 @@
+import Divider from '@mui/material/Divider'
+import Typography from '@mui/material/Typography'
+import Box from '@mui/material/Box'
+import { ReactElement } from 'react'
+import Stack from '@mui/material/Stack'
+
+import FormGroup from '@mui/material/FormGroup'
+import FormControlLabel from '@mui/material/FormControlLabel'
+import Checkbox from '@mui/material/Checkbox'
+import IconButton from '@mui/material/IconButton'
+import Radio from '@mui/material/Radio'
+import RadioGroup from '@mui/material/RadioGroup'
+import CloseRoundedIcon from '@mui/icons-material/CloseRounded'
+import { ClearAllButton } from './ClearAllButton'
+
+interface Props {
+  handleClose?: () => void
+  handleChange?: (e) => void
+  sortSetting: string
```

Comment:
Change type so that we limit what strings can be passed in.
```suggestion
  sortSetting: 'date' | 'duration'
```